undebt/pattern/interface.py

```diff
@@ -74,10 +76,35 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        return _load_module(path)
+
     pattern_name = os.path.splitext(os.path.basename(path))[0]
     return imp.load_source(pattern_name, path)
 
 
+def _load_module(full_name):
+    try:
+        return importlib.import_module(full_name)
+    except ImportError as e:
+        raise e
+    finally:
+        # Unshim path
+        sys.path = sys.path[1:]
```

Comment:
I wouldn't change the behaviour that standard python developers expect.  If my module isn't importable using `python -c 'import foo'` I wouldn't expect it to be importable in undebt.

Also you've got a potential bug here if importing has a side-effect of modifying sys.path.  You're better off just leaving sys.path alone :)